offscreen/offscreen.html

```diff
@@ -7,5 +7,6 @@
   <body>
     <script type="text/javascript" src="./load-offscreen.js"></script>
     <iframe src="./trezor-iframe.html"></iframe>
+    <iframe src="./ledger-iframe.html"></iframe>
```

Comment:
Would this security hardening work?
```suggestion
    <iframe sandbox="" src="./ledger-iframe.html"></iframe>
```
Or does it clash with the rest of the logic introduced in this PR? Could be a nice touch. Weird, because the origin of the content of the iframe is at a similar trust level of the code that loads it, but if such isolation costs us nothing, it could only add security without giving up on anything.